docs/source/en/model_doc/segformer.mdx

```diff
@@ -36,13 +36,14 @@ The figure below illustrates the architecture of SegFormer. Taken from the [orig
 
 <img width="600" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/segformer_architecture.png"/>
 
-This model was contributed by [nielsr](https://huggingface.co/nielsr). The original code can be found [here](https://github.com/NVlabs/SegFormer).
```

Comment:
@sgugger I have added some minor notes to the model doc. Would appreciate 👀.

Reply:
@sgugger a friendly nudge.

Reply:
@gante I have modified the model doc a bit to reflect on two things primarily:

* A repository that shows how to perform off-the-shelf inference and fine-tuning with the TF SegFormer model.
* An interactive demo hosted on Spaces. 

@deep-diver and I contributed to that effort equally. 

Could you please review the modifications I made to the doc once you get a moment?